ios/native/AppDelegate.m

```diff
@@ -35,7 +35,14 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
                                                  initialProperties:nil
                                                      launchOptions:launchOptions];
     rootView.backgroundColor = [[UIColor alloc] initWithRed:1.0f green:1.0f blue:1.0f alpha:1];
-
+    
+    // Adding here for when we migrate to native RN DM support
```

Comment:
Surely we've done that already in this PR?

Reply:
Yes good spot need to re instate this now! From an iOS perspective do these colours match what we do on Backpack iOS

Reply:
Wondering if we actually need this at all, given that the app seems to work fine and respect the correct colours already without it?

I had forgotten to address this so wondering if its still relevant